README.md

```diff
@@ -60,6 +60,7 @@ An [init tag](https://github.com/aws/aws-for-fluent-bit#using-the-init-tag) is d
 * [Sematext Logs](examples/fluent-bit/sematext)
 * [Logstash](examples/fluent-bit/logstash)
 * [Elastic Cloud](examples/fluent-bit/elastic-cloud)
+* [Grafana Loki monitoring](examples/fluent-bit/grafana-loki)
```

Comment:
If this is for grafana cloud, should we just title it that? Is this what grafana users expect?

If they see later:
- "Grafana Loki monitoring"
- "Amazon Managed Grafana"

I guess it will be clear? yesterday we said we'd call it grafana cloud- so why'd you change your thinking?

Reply:
Also this link does not work- the file path is grafana-cloud

Reply:
forgot to update this file.